src/index.md

```diff
@@ -21,4 +28,13 @@ If you would like to make contributions to this documentation site, it's hosted
 
 * Talk to current developers in Discord.
 * Check [the list of issues](https://github.com/space-wizards/space-station-14/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) for anything you can work on.
-* Reading through [Acronyms & Common Nomenclature](en/general-development/codebase-info/acronyms-and-nomenclature.md) as well as the other articles on this wiki in `Tips` or `Setup` will likely help you understand what is happening.
\ No newline at end of file
+* Reading through [Acronyms & Common Nomenclature](en/general-development/codebase-info/acronyms-and-nomenclature.md) as well as the other articles on this wiki in `Tips` or `Setup` will likely help you understand what is happening.
+
+### What repositories are part of Space Station 14?
+* [Space Station 14](https://github.com/space-wizards/space-station-14/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) is the main codebase. It has the most priority issues. 
+* [Robust Toolbox](https://github.com/space-wizards/RobustToolbox/issues) is the custom game engine.
+
+There are other repositories that do not have as many priority issues to work on. If you are still curious about them, they are:
+- [Network](https://github.com/space-wizards/SpaceWizards.Lidgren.Network/issues)
+- [Server Launcher](https://github.com/space-wizards/SS14.Launcher/issues) 
+- [Other](https://github.com/space-wizards) 
```

Comment:
There's significantly more than this (e.g. RSIEdit, Arch) so better to just say there's others under space-wizards